specification/web/resource-manager/Microsoft.Web/stable/2023-12-01/CommonDefinitions.json

```
@@ -3624,7 +3624,7 @@
                 "blobContainer"
               ],
               "x-ms-enum": {
-                "name": "storageType",
+                "name": "functionsDeploymentStorageType",
```

nit, camelcase for the name
```suggestion
                "name": "FunctionsDeploymentStorageType",
```